static/app/views/monitors/components/detailsSidebar.tsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import {SectionHeading} from 'sentry/components/charts/styles';
```

Comment:
Why is this in charts 😬

Reply:
Lol yea, seems like most of the times this is used in charts but definitely not only charts, should probably just bring it out into components.